packages/@react-spectrum/searchwithin/test/SearchWithin.test.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import {act, render} from '@testing-library/react';
+import {Item, Picker} from '@react-spectrum/picker';
+import {Provider} from '@react-spectrum/provider';
+import React from 'react';
+import {SearchField} from '@react-spectrum/searchfield';
+import {SearchWithin} from '../src';
+import {theme} from '@react-spectrum/theme-default';
+import {triggerPress} from '@react-spectrum/test-utils';
+import {typeText} from '@react-spectrum/test-utils';
+
+let defaultProps = {
+  label: 'Test'
+};
+
+function renderSearchWithin(props = {}, searchFieldProps = {}, pickerProps = {}) {
+  return render(
+    <Provider theme={theme}>
+      <SearchWithin {...defaultProps} {...props}>
+        <SearchField placeholder="Search" {...searchFieldProps} />
+        <Picker defaultSelectedKey="all" {...pickerProps}>
+          <Item key="all">All</Item>
+          <Item key="campaigns">Campaigns</Item>
+          <Item key="audiences">Audiences</Item>
+          <Item key="tags">Tags</Item>
+        </Picker>
+      </SearchWithin>
+    </Provider>  
+  );
+}
+
+describe('SearchWithin', function () {
+  beforeAll(function () {
+    jest.useFakeTimers();
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+    act(() => jest.runAllTimers());
+  });
+
+  afterAll(function () {
+    jest.restoreAllMocks();
+  });
+
+  it('renders correctly', function () {
+    let {getAllByText, getByRole, getByPlaceholderText} = renderSearchWithin();
+
+    let searchfield = getByPlaceholderText('Search');
```

Comment:
probably better to get by role whenever possible, that's very unlikely to change
you should be able to get it with the role 'textbox'
I looked back at our textfield tests, they need some updating, haha
I've been trying to follow the priority recommendation https://testing-library.com/docs/queries/about#priority

Reply:
Unfortunately, they might need explicit `type="text"` to work with getByRole. I'm getting a `TestingLibraryElementError: Unable to find an accessible element with the role "textbox"`.

Relevant issues: 
https://github.com/testing-library/react-testing-library/issues/606
https://github.com/testing-library/dom-testing-library/issues/500

Reply:
ah, that's a shame, well, we can still move over to the label which will verify that the input is labeled implicitly
`let searchfield = getByLabelText('Test', {selector: 'input'});`
https://testing-library.com/docs/queries/bylabeltext/#api